lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8235,6 +8248,19 @@ static bool isUsingMacroName(clang::SourceManager &SM,
   return content == MacroName;
 }
 
+/// Utility function to import clang attributes from a source swift decl to synthesized swift decl.
+void ClangImporter::Implementation::ImportAttributesFromClangDeclToSynthesizedSwiftDecl(Decl *sourceDecl, Decl* synthesizedDecl)
```

Comment:
Minor: the Swift code style assumes `lowerCasedFunctionNames`, could you please rename this function?